cy_canvas/reducers/cyElements.js

```diff
@@ -220,6 +235,17 @@ const cyElements = (state, action) => {
         action: action.action
       }))
 
+    case "DELETE_ELEMENTS":
+      const remain = state.filter(t => 
+        !action.elements.map(getId).includes(t.data.id)
```

Comment:
It's quite elegant to write only one line of code to solve this problem, but every time a "t" is pass to filter function, "action.elements.map(getId)" will be called once. However, this won't be a problem since we won't delete a great amount of node at the same time supposedly.